doc/releases/changelog-dev.md

```diff
@@ -20,11 +20,13 @@
   [(#8520)](https://github.com/PennyLaneAI/pennylane/pull/8520)
   [(#8515)](https://github.com/PennyLaneAI/pennylane/pull/8515)
   [(#8516)](https://github.com/PennyLaneAI/pennylane/pull/8516)
+  [(#8558)](https://github.com/PennyLaneAI/pennylane/pull/8558)
   [(#8534)](https://github.com/PennyLaneAI/pennylane/pull/8534)
 
   - :class:`~.QSVT`
   - :class:`~.AmplitudeEmbedding`
   - :class:`~.AllSinglesDoubles`
+  - :class:`~.GateFabric`
   - :class:`~.IQPEmbedding`
```

Comment:
I just realized that it looks funny t his way:
<img width="831" height="314" alt="image" src="https://github.com/user-attachments/assets/60005415-2ab4-47d8-a8a2-f174d2515143" />
I think probably we can try put the hashtag and link right after their correponding modified class?

Reply:
Yeah it's not a big deal lol feel free to ignore